src/pages/core/standard-library/math/integers.mdx

```diff
@@ -0,0 +1,50 @@
+---
+tags: ["core", "math"]
+---
+
+import { Callout } from "nextra/components";
+
+# Integers
+
+CosmWasm offers integer types starting at 64 bit precision, all the way up to
+512 bits.
+
+Reasoning behind wrapping the native 64 bit type is the interaction with
+JavaScript. A `u64` would normally serialize into a regular integer field in
```

Comment:
This is not only JavaScript specific. Other JSON parsets like jq behave the same (all numbers are stired as double). That being said, JavaScript is certainly the most important for us.